### PR TITLE
Add Dart option docs for `considerInAppFramesByDefault`

### PR DESCRIPTION
**Pull request by ueman**
Relates to https://github.com/getsentry/sentry-dart/pull/482

cc @marandaneto

### CODE REVIEW

File: src/platforms/common/configuration/options.mdx [hunk 1]
```diff
@@ -182,6 +182,17 @@ When set to `true`, the SDK will send session events to Sentry. This is supporte
 
 </ConfigKey>
 
+<ConfigKey name="consider-frames-in-app-by-default" supported={["dart"]}>
```
Comment (marandaneto) [inline src/platforms/common/configuration/options.mdx:185]: if we agree on renaming, `considerInAppFramesByDefault`, we need to fix key names and param names here

Comment (ueman) [reply src/platforms/common/configuration/options.mdx:185]:
```suggestion
<ConfigKey name="consider-in-app-frames-by-default" supported={["dart"]}>
```

File: src/platforms/common/configuration/options.mdx [hunk 2]
```diff
@@ -182,6 +182,17 @@ When set to `true`, the SDK will send session events to Sentry. This is supporte
 
 </ConfigKey>
 
+<ConfigKey name="consider-in-app-frames-by-default" supported={["dart"]}>
+
+Configures whether stack trace frames are considered as in app frames by default.
+You can use this to essentially make `inAppIncludes` or `inAppExcludes` an allow or deny list.
+This value is only used if Sentry can not find the origin of the frame.
```
Comment (PeloWriter) [inline src/platforms/common/configuration/options.mdx:189]:
```suggestion
This value is used only if Sentry can not find the origin of the frame.
```

File: src/platforms/common/configuration/options.mdx [hunk 3]
```diff
@@ -182,6 +182,17 @@ When set to `true`, the SDK will send session events to Sentry. This is supporte
...
+
+- If `considerFramesInAppByDefault` is true you only need to maintain `inAppExcludes`.
+- If `considerFramesInAppByDefault` is false you only need to maintain `inAppIncludes`.
```
Comment (PeloWriter) [inline src/platforms/common/configuration/options.mdx:192]:
```suggestion
- If `considerFramesInAppByDefault` is set to `true`, you only need to maintain `inAppExcludes`.
- Conversely, if `considerFramesInAppByDefault` is set to `false`, you only need to maintain `inAppIncludes`.
```

**Review by PeloWriter — APPROVED** [review]

File: src/platforms/common/configuration/options.mdx (hunk 3 above)
Comment (ueman) [reply src/platforms/common/configuration/options.mdx:192]: That's a nice change.

File: src/platforms/common/configuration/options.mdx [hunk 4]
```diff
@@ -182,6 +182,17 @@ When set to `true`, the SDK will send session events to Sentry. This is supporte
...
 </ConfigKey>
 
+<ConfigKey name="consider-in-app-frames-by-default" supported={["dart"]}>
+
+Configures whether stack trace frames are considered as in app frames by default.
+You can use this to essentially make `inAppIncludes` or `inAppExcludes` an allow or deny list.
+This value is used only if Sentry can not find the origin of the frame.
+
+- If `considerFramesInAppByDefault` is set to `true`, you only need to maintain `inAppExcludes`.
+- Conversely, if `considerFramesInAppByDefault` is set to `false`, you only need to maintain `inAppIncludes`.
```
Comment (marandaneto) [inline src/platforms/common/configuration/options.mdx:192]:
```suggestion
- If `considerInAppFramesByDefault` is set to `true`, you only need to maintain `inAppExcludes`.
- Conversely, if `considerInAppFramesByDefault` is set to `false`, you only need to maintain `inAppIncludes`.
```

**Review by marandaneto — APPROVED** [review]
LGTM

**Pull request merged**